### ISSUE TITLE
Cant Open Edge

### ISSUE THREAD

**Issue by josegalf** [igorskyflyer/vscode-open-in-browser#10]
The extension doesn't recognize Edge browser and it doesn't launch it even if it is the default browser

**Comment by igorskyflyer** [comment]
Correct, because older builds of Edge didn't support opening of local files like the more current ones do, I will add that feature and publish a long-awaited update for the extension. 😄
